app/models/user.rb

```diff
@@ -350,7 +302,7 @@ class User < ApplicationRecord
 
   after_create_commit :send_welcome_notification
 
-  after_commit :sync_users_settings_table, :sync_users_notification_settings_table, on: %i[create update]
```

Comment:
> Since we remove the sync_users_settings_table method completely, there is no users_setting record being associated when a user is created.

🤦🏾‍♀️  🤦🏾‍♀️  🤦🏾‍♀️  🤦🏾‍♀️  Thank you 🙏🏾